server/game/cards/10-TEL/ThoseWhoServe.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard');
+const {CardTypes, Durations, Phases, PlayTypes} = require('../../Constants');
+
+class ThoseWhoServe extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Reduce the cost of your characters by 1 this phase',
+            phase: Phases.Dynasty,
+            effect: 'reduce the cost of their characters by 1 this phase',
+            gameAction: ability.actions.playerLastingEffect({
+                duration: Durations.UntilEndOfPhase,
+                effect: ability.effects.reduceCost({
+                    // RRG does not allow characters to be played from hand in dynasty, but does not use the term "cannot" for this,
+                    // so future card effects could nullify this restriction. In this case conflict characters would work with Those Who Serve.
+                    playingTypes: [PlayTypes.PlayFromHand, PlayTypes.PlayFromProvince],
```

Comment:
Because the card doesn't specify a playingType, there shouldn't be one here.  However, making this change won't pass tests until #3602 is merged into TEL

Reply:
Ok, I'll update this PR later then. I've seen devel being merged into a card branch on a few PRs, would that also be appropriate here once #3602 is merged?

Reply:
Fixed.